src/models/userDao.js

```diff
@@ -0,0 +1,15 @@
+const { AppDataSource } = require("./dataSource")
+
+const signUp = async (email, password, nickname, phoneNumber) => {
+    await AppDataSource.query(`INSERT INTO users (email, password, nickname, phone_number) VALUES (?, ?, ?, ?)`, [email, password, nickname, phoneNumber])
+}
+
+const findByEmail = async (email) => {
+    const [existingEmail] = await AppDataSource.query(`SELECT email FROM users WHERE email = (?)`, [email])
+    return existingEmail
+}
+
```

Comment:
```suggestion

const signUp = async (email, password, nickname, phoneNumber) => {
    await AppDataSource.query(`
      INSERT INTO users (
        email, 
        password, 
        nickname, 
        phone_number
      ) VALUES (?, ?, ?, ?)
    `, [email, password, nickname, phoneNumber])
}

const findByEmail = async (email) => {
    const [existingEmail] = await AppDataSource.query(`
      SELECT email 
      FROM users 
      WHERE email = ?
      `, [email])
    return existingEmail
}

```
1. SQL 문 분리해서, 가독성을 좀 더 높여주시면 좋을 것 같습니다 :)
2. WHERE절 이후에 있는 ?에 소괄호를 입력하신 이유가 따로 있을까요? 없으면 삭제하면 될 것 같고, 이유가 있으면 알려주세요!